src/main/java/ch/njol/skript/expressions/ExprStatistics.java

```diff
@@ -0,0 +1,232 @@
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.bukkitutil.EntityUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Material;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Statistic;
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Player Statistics")
+@Description({
+	"Returns the value of one or more statistics for one or more players.",
+	"Statistics are countable data for every player, which is tracked by the server.",
+	"You can find more info on the <a href=\"https://minecraft.fandom.com/wiki/Statistics\">Minecraft Wiki</a> and a list of statistic names on the " +
+		"<a href=\"https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Statistic.html\">Spigot Javadocs</a>.",
+})
+@Examples({
+	"# Examples:",
+	"# \t/statistics AwesomePlayerName MINE_BLOCK sand itemtype",
+	"# \t/statistics AwesomePlayerName KILL_ENTITY zombie entity",
+	"# \t/statistics AwesomePlayerName CHEST_OPENED",
+	"command /statistics <offlineplayer> <string> [<text>] [<text>]:",
+	"\tusage: Usage: /statistics <player> <statistic = string> [kind e.g. cow, stone] [type = entity/itemtype]",
+	"\ttrigger:",
+	"\t\tif arg-3 is set:",
+	"\t\t\tif arg-4 = \"entity\":",
+	"\t\t\t\tset {_kind} to arg-3 parsed as entitytype",
+	"\t\t\telse if arg-4 = \"itemtype\":",
+	"\t\t\t\tset {_kind} to arg-3 parsed as itemtype",
+	"\t\t\telse:",
+	"\t\t\t\tsend \"You need to specify type: entity/itemtype\"",
+	"\t\t\t\tstop",
+	"",
+	"\t\t\tif {_kind} is set:",
+	"\t\t\t\tsend statistic arg-2 for {_kind} of arg-1 to player",
+	"\t\telse:",
+	"\t\t\tsend statistic arg-2 of arg-1 to player",
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("MC 1.15+ (offlineplayers)")
+public class ExprStatistics extends SimpleExpression<Long> {
+
+	private static final String PLAYER_PATTERN = Skript.isRunningMinecraft(1, 15) ? "%-offlineplayers%" : "%-players%";
```

Comment:
The players Expression is not defined as nullable. Is there a reason these are nullable and not default?